src/docs/product/accounts/pricing.mdx

```diff
@@ -54,7 +54,7 @@ If your projects have a high volume of events, you can control how many errors S
 
 ## FAQ
 
-**Is there detailed information for each plan?**
+#### Is there detailed information for each plan?
```

Comment:
Shouldn't this be `###` if FAQ has `##`?

Reply:
Structurally, yes. But in this case, the change was made to fix a bolding issue, so an H4 is fine. Thanks for reviewing.